pkg/artifacts/signable.go

```diff
@@ -116,6 +116,7 @@ func ExtractOCIImagesFromResults(tr *v1beta1.TaskRun, logger *zap.SugaredLogger)
 	urlSuffix := "IMAGE_URL"
 	digestSuffix := "IMAGE_DIGEST"
 	for _, res := range tr.Status.TaskRunResults {
+		// Note(rgreinho): using HasSuffix should be enough.
 		if strings.HasSuffix(res.Name, urlSuffix) || res.Name == urlSuffix {
```

Comment:
i think you're correct that [HasSuffix is sufficient!](https://go.dev/play/p/IlhbHza049E) we could probably remove the second check in both places :)

Reply:
Addressed ✅